atlasdb-api/versions.lock

```diff
@@ -1,21 +1,21 @@
 {
     "compileClasspath": {
         "com.fasterxml.jackson.core:jackson-annotations": {
-            "locked": "2.6.7",
+            "locked": "2.8.6",
             "transitive": [
                 "com.fasterxml.jackson.core:jackson-databind",
                 "com.palantir.atlasdb:atlasdb-commons",
                 "com.palantir.atlasdb:timestamp-api"
             ]
         },
         "com.fasterxml.jackson.core:jackson-core": {
-            "locked": "2.6.7",
+            "locked": "2.8.6",
```

Comment:
I don't think we can bump Jackson above 2.6.x in the `atlasdb-api` or other externally consumed modules as it will cause downstream version conflicts. If we have to bump Jackson for DW, we will have to confine it to the `timelock-server` module.